_posts/en/newsletters/2020-07-22-newsletter.md

```diff
@@ -0,0 +1,185 @@
+  of their multi-account desktop multisig vault application. Lily Wallet creates
+  2-of-3 bech32 addresses and uses [HWI][topic hwi], HD wallets, and PSBT to transact.
+
+- **Zap 0.7.0 Beta released:** [Zap 0.7.0 Beta][zap 0.7.0] adds [spontaneous payments][topic
+  spontaneous payments], [payment probes][news30 probing], [multipath
+  payments][topic multipath payments], and [hold invoices][topic hold invoices]
+
+- **BTCPay Server 1.0.5.0 implements various standards:**
+  BTCPay Server 1.0.5.0 adds [BIP78][], BIP21, and additional [PSBT][topic psbt]
+  support.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.10.4][] FIXME:harding (windows users only)
+
+- [Bitcoin Core 0.20.1rc1][Bitcoin Core 0.20.1] FIXME:harding
+
+- [C-Lightning 0.9.0rc2][C-Lightning 0.9.0] FIXME:harding
+
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #19109][] introduces further improvements to
+  [transaction-origin privacy][transaction origin wiki]. Building on
+  [Bitcoin Core #18861][bitcoin 18861 newsletter], this patch adds a
+  per-peer rolling bloom filter to track which transactions were recently
+  announced to the peer. When a peer requests a transaction, this upgrade
+  checks the filter before fulfilling the request and relaying the
+  transaction to prevent spy nodes from learning about the exact contents
+  of our mempool or the precise timing of when a transaction was received.
+
+- [Bitcoin Core #16525][] updates several RPCs to return transaction
+  version numbers as unsigned integers rather than signed integers.
+  Early versions of Bitcoin interpreted version numbers as signed
+  integers (and
+  [some][35e79ee733fad376e76d16d1f10088273c2f4c2eaba1374a837378a88e530005]
+  transactions in the chain did actually set negative version numbers),
+  but the introduction of [BIP68][] nSequence enforcement for
+  transactions with version 2 or higher specified that it would match
+  against transaction nVersion fields cast to unsigned integers.  It's
+  expected that any other future soft forks that use transaction
+  versions will specify the same rule, and so it makes sense for RPCs to
+  always return unsigned numbers for transaction versions.
+
+- [C-Lightning #3809][] adds support to C-Lightning for effective
+  sending of [multipath payments][topic multipath payments]---payments
+  which are split into several parts, with each part routed using a
+  different path.  In brief, the algorithm C-Lightning uses is to split
```

Comment:
"uses is to split a payment" -> maybe "uses splits payments" or "C-Lightning's algorithm splits"